azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageTypeDaoImpl.java

```diff
@@ -59,33 +82,43 @@ public int createImageType(ImageType imageType) {
       log.info("Created image type id :"+imageTypeId);
     } catch (SQLException e) {
       log.error("Unable to create the image type metadata", e);
+      throw new ImageMgmtDaoException("Exception occurred while creating image type metadata");
     }
     return imageTypeId;
   }
 
   @Override
-  public Optional<ImageType> getImageTypeByType(String type) {
+  public Optional<ImageType> getImageTypeByName(String name) {
     final FetchImageTypeHandler fetchImageTypeHandler = new FetchImageTypeHandler();
     List<ImageType> imageTypes = new ArrayList<>();
     try {
       imageTypes = this.databaseOperator
-          .query(FetchImageTypeHandler.FETCH_IMAGE_TYPE_BY_TYPE, fetchImageTypeHandler, type);
+          .query(FetchImageTypeHandler.FETCH_IMAGE_TYPE_BY_NAME, fetchImageTypeHandler, name);
+      //
```

Comment:
Nit: remove empty comment

Reply:
Thanks for pointing this. Added comment.